nectar/src/main.rs

```diff
@@ -13,7 +13,7 @@
 #![cfg_attr(not(test), warn(clippy::unwrap_used))]
 #![forbid(unsafe_code)]
 #![recursion_limit = "512"]
-#![type_length_limit = "1944624"]
+#![type_length_limit = "7743004"]
```

Comment:
💥

Reply:
Yeah, don't know what is going on here :(